charts/astronomer/values.yaml

```diff
@@ -259,7 +259,7 @@ registry:
   # k8s secret name for registry to use to authenticate to houston when sending notifications
   authHeaderSecretName: ~
   replicas: 1
-  podLabels: {}
+  podLabels: {} # This almost certainly does nothing. The right value is astronomer.podLabels
```

Comment:
@andriisoldatenko do you know anything about this? (2/2)

Reply:
```
./charts/astronomer/templates/registry/registry-statefulset.yaml
37:{{- with .Values.podLabels }}
```
?

Reply:
@andriisoldatenko this is `astronomer.commander.podLabels`

Reply:
See https://github.com/astronomer/astronomer/blob/18770565233bac830319653df524b3e706ddbc67/tests/test_registry_statefulset.py#L111-L120 for `astronomer.podLabels` test in registry sts